examples/api/lib/widgets/shortcuts/callback_shortcuts.0.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for [CallbackShortcuts].
```

Comment:
Move this below the imports (to avoid a lint), and convert to a document comment, so that the link is enabled back to the originating class for IDE users.

Reply:
Fixed